courses/machine_learning/deepdive2/structured/labs/4c_keras_wide_and_deep_babyweight.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>nit: whitespace in line 11 and line 18</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Thanks</p>